src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -185,79 +201,293 @@ sub-01_run-2_T1w.json
+| Homogeneous (flat) T1 image by MP2RAGE                 | UNIT1     | (M) --> In arbitrary units (a.u.). UNIT1 maps are REQUIRED to use this suffix irrespective of the method they are related to.  _Can be generated from:_ `MP2RAGE`                                                                                                                                                                                                                                                                                                                                                                                                                             |
+| Longutidunal relaxation in rotating frame (T1 rho) map | T1Rmap    | (M) --> In seconds (s). T1-rho maps are REQUIRED to use this suffix irrespective of the method they are related to. _Can be generated from:_ N/A                                                                                                                                                                                                                                                                                                                                                                                                                                         |
+| Myelin water fraction map                              | MWFmap    | (M) --> In percentage (%). MWF maps are REQUIRED to use this suffix irrespective of the method they are related to.  _Can be generated from:_ `MESE`                                                                                                                                                                                                                                                                                                                                                                                                                                      |
+| Combined PD/T2 map                                     | PDT2map   | (M) --> In arbitrary units (a.u.). Combined PD/T2 maps are REQUIRED to use this suffix irrespective of the method they are related to. N/A                                                                                                                                                                                                                                                                                                                                                                                                                    |                                                                                                                                                                                                                                                                                                                                                                                                               |
```

Comment:
Need help from eg Tom Nichols to define this.

Might want to keep `suffix` as the current one which is `PDT2`.

Could put this either in the conventional acquistion section or in derived map - but we aren't sure what type of scan it is and who uses it!